go.mod

```diff
@@ -10,6 +10,8 @@ require (
 	github.com/huandu/xstrings v1.2.0 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/mattermost/mattermost-server v0.0.0-20191017141203-48c06e9bce3b
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
```

Comment:
just wondering why these weren't in before, tidy-ed?

Reply:
Hmm, uncertain -- I'll push a `go mod tidy` to fix this for now, but I note they come back with a `make clean test`.